### PR TITLE
Add unit test to verify fix for #754

### PR DESCRIPTION
**Pull request by mrfelton**
Add unit test to verify fix for #754

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by superkhau** [comment]
LGTM

**Comment by superkhau** [comment]
@slnode test please

**Comment by superkhau** [comment]
LGTM

File: test/include.test.js [hunk 1]
```diff
@@ -427,6 +427,30 @@ describe('include', function () {
     });
   });
 
+  // https://github.com/strongloop/loopback-datasource-juggler/issues/754.
+  it('should save related items separately', function(done) {
+    var self = this;
```
Comment (superkhau) [inline test/include.test.js]: Unused, remove please.

**Comment by superkhau** [comment]
@rmg Why is loopback-example-app still a dependent (and failing)? I thought we removed this already from CI.

**Comment by superkhau** [comment]
@mrfelton I made a few minor edits to your PR. Otherwise, LGTM and thanks for the contribution! ;)

**Comment by superkhau** [comment]
Oops, I merged in the wrong branch. See https://github.com/strongloop/loopback-datasource-juggler/pull/828

**Pull request merged**